capstone/src/main/java/com/google/univiz/api/SearchResource.java

```diff
@@ -1,7 +1,15 @@
 package com.google.univiz.api;
 
+import java.util.List;
+
 /**
  * The SearchResoure interface is responsible for all data management/processing specific for the
  * search feature of the UniViz webapp.
  */
-public interface SearchResource {}
+public interface SearchResource {
+  /**
+   * Take a list of suggestion data and returns a list of Suggestions objects to the caller. This
+   * object contains all data relevant to the Search feature.
+   */
+  List<SearchData> getSearchSuggestions(List<SuggestionData> collegeNames);
```

Comment:
IIUC `SearchResource.getSearchSuggestions` will be invoked from a `SearchServlet`.  How will the servlet produce `SuggestionData`? Should this method take a String argument, instead?

Reply:
OMG yes-I confused myself here too for this PR and the other one chained with this 

that makes a lot more sense-will update!